packages/components/PartyInfo.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Text } from '@chakra-ui/react';
+
+interface IPartyInfo {
+  candidate?: string;
+  vice?: string;
+  partido?: string;
+}
+
+export default function PartyInfo({ candidate, vice, partido }: IPartyInfo) {
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      border="2px solid black"
+      w="35vw"
+      h="35vh"
+      justifyContent="center"
+      alignItems="left"
+    >
+      <Box display="flex" marginLeft="10%" py="20px">
```

Comment:
Why is a marginLeft here? We should avoid margins, shouldn't this be a Box with 100% width?